.gitignore

```diff
@@ -0,0 +1,146 @@
+# Dependencies
+node_modules/
```

Comment:
Something a little off here - I can see the node_modules have been committed to the repository, we'll need to remove those